src/main.js

```diff
@@ -1,18 +1,39 @@
 // Create variables targetting the relevant DOM elements here 👇
 
 
+
+var coverImg = document.querySelector('.cover-image');
+var title = document.querySelector('.cover-title');
+var tagline1 = document.querySelector('.tagline-1');
+var tagline2 = document.querySelector('.tagline-2');
```

Comment:
Hey lets refactor these variable names to descriptor1 and descriptor2 seeing as how these variables are only housing the single descriptor vs. the whole tagline.